README.md

```diff
@@ -15,6 +15,9 @@ npm install screwdriver-screwdriver-coverage-sonar
 npm test
```

Comment:
Tiff, can you also update build links etc. in the readme? I forgot to do it last time. Otherwise LGTM 👍